### PR TITLE
dev-libs/hyperscan: warn rspamd users

### PR DESCRIPTION
**Pull request by arkamar**
see commit message for more info.

### CODE REVIEW

File: mail-filter/rspamd/rspamd-3.4-r3.ebuild [hunk 1]
```diff
@@ -54,7 +54,7 @@ RDEPEND="${LUA_DEPS}
 		virtual/blas
 		virtual/lapack
 	)
-	cpu_flags_x86_ssse3? ( dev-libs/hyperscan )
+	cpu_flags_x86_ssse3? ( ~dev-libs/hyperscan-5.4.0 )
```
Comment (thesamesam) [inline mail-filter/rspamd/rspamd-3.4-r3.ebuild:57]: Add a comment above please.

Comment (arkamar) [reply mail-filter/rspamd/rspamd-3.4-r3.ebuild:59]: done

**Comment by candrews** [comment]
How will the resulting downgrade if this PR is merged impact users who have already done the workaround of removing /var/lib/rspamd/*.hs* files?

**Comment by arkamar** [comment]
They will have to do the same workaround again for downgrading. But I don't have better solution for now. Any idea?

**Comment by arkamar** [comment]
But maybe it is not neccesary to rush with this right now, becuase `dev-libs/hyperscan-5.4.1` is under `~` keword?

**Comment by arkamar** [comment]
I am proposing another solution, let's print warning message from `hyperscan` when `rspamd` is installed.

**Comment by arkamar** [comment]
I can limit it more, if `/var/lib/rspamd/*.hs*` files are present. But it smells a little bit like dark magic :)

**Review by juippis — APPROVED** [review]
Sure why not, better than nothing I guess!

**Pull request closed without merging**